sdcm/utils/sstable/load_utils.py

```diff
@@ -175,9 +163,49 @@ def validate_load_and_stream_status(cls, node, system_log_follower,
                 break
 
         assert load_and_stream_started, f'Load and stream has not been run on the node {node.name}'
+        return load_and_stream_status
+
+    @classmethod
+    @timeout_decor(
+        timeout=60,
+        allowed_exceptions=(AssertionError,),
+        message="Waiting for load_and_stream completion message to appear in logs")
+    def wait_for_load_and_stream_finish(cls, node, system_log_follower,
+                                        keyspace_name='keyspace1', table_name='standard1'):
+        load_and_stream_messages = list(system_log_follower)
+        assert load_and_stream_messages, f"Load and stream wasn't finished on the node {node.name}"
+        LOGGER.debug("Found load_and_stream messages: %s", load_and_stream_messages)
+
+        load_and_stream_status = "n/a"
+        for line in load_and_stream_messages:
+            load_and_stream_done = re.search(cls.LOAD_AND_STREAM_DONE_EXPR.format(keyspace_name, table_name), line)
+            if load_and_stream_done:
+                load_and_stream_status = load_and_stream_done.groups()[0]
+                break
+
         assert load_and_stream_status == 'succeeded', \
             f'Load and stream status  on the node {node.name} is "{load_and_stream_status}". Expected "succeeded"'
 
+    @classmethod
+    def validate_load_and_stream_status(cls, node, system_log_follower,
+                                        keyspace_name='keyspace1', table_name='standard1'):
+        """
+        Validate that load_and_stream was started and has been completed successfully.
+        Search for a messages like:
+            storage_service - load_and_stream: ops_uuid=c06c76bb-d178-4e9c-87ff-90df7b80bd5e, ks=keyspace1,
+             table=standard1, target_node=10.0.3.31, num_partitions_sent=45721, num_bytes_sent=24140688
+
+            storage_service - Done loading new SSTables for keyspace=keyspace1, table=standard1, load_and_stream=true,
+            primary_replica_only=false, status=succeeded
+
+        Starting with the Scylla 4.6 version the prefix becomes 'sstables_loader' instead of
+        the 'storage_service' one.
+        """
+        load_and_stream_status = cls.wait_for_load_and_stream_start(node, system_log_follower,
+                                                                    keyspace_name, table_name)
+        if not load_and_stream_status:
+            cls.wait_for_load_and_stream_finish(node, system_log_follower, keyspace_name, table_name)
+
```

Comment:
Scenario where `load_and_stream_status` is None or empty is not possible 
```suggestion
        cls.wait_for_load_and_stream_finish(node, system_log_follower, keyspace_name, table_name)
```

Reply:
Why it's not possible?
If load was started but not finished, we need to wait for finish

Reply:
`timeout_decor` will raise in such case

Reply:
You are right. Fixed

Reply:
No it, not correct. My solution was correct. Because of it's `load_and_stream_status`, that means the load was started but not completed. Return it back

Reply:
load_and_stream_status is 'n/a' by default

Reply:
fixed